web/packages/design/src/ThemeProvider/index.tsx

```diff
@@ -39,6 +39,24 @@ function themePreferenceToTheme(themePreference: Theme) {
   return themePreference === Theme.LIGHT ? lightTheme : darkTheme;
 }
 
+// because unspecified can exist but only used as a fallback and not an option,
+// we need to get the current/next themes with getPrefersDark in mind.
+// TODO (avatus) when we add user settings page, we can add a Theme.SYSTEM option
+// and remove the checks for unspecified
+export function getCurrentTheme(currentTheme: Theme): Theme {
+  if (currentTheme === Theme.UNSPECIFIED) {
+    return getPrefersDark() ? Theme.DARK : Theme.LIGHT;
```

Comment:
how do i test this?  i created a new user, and it defaulted to light theme (using brave)

<img width="1380" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/4d5f313c-9c03-465e-ac14-1df4f2d36895">

<img width="727" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/f57ef095-c446-43ae-85fb-14d368309fb6">

Reply:
I wasn't able to reproduce this first click not working in incognito or normal mode